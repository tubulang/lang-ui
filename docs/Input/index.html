<!DOCTYPE html><html lang="en"><head><meta charSet="utf-8"/><meta http-equiv="x-ua-compatible" content="ie=edge"/><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/><style data-href="/lang-ui/styles.813b51114db8f0dc1b27.css">@import url(https://fonts.googleapis.com/css?family=Inconsolata&display=swap);@import url(https://fonts.googleapis.com/css?family=Source+Sans+Pro&display=swap);.tubulang-alert{padding:20px;border-radius:3px;color:rgba(0,0,0,.65)}.tubulang-alert-positive{background-color:#52c41a}.tubulang-alert-info{background-color:#1890ff}.tubulang-alert-negative{background-color:#ff4d4f}.tubulang-alert-warning{background-color:#faad14}.tubulang-input{padding:5px 7px;height:20px;border:1px solid #d9d9d9;border-radius:2px;outline:0}.tubulang-input:active,.tubulang-input:focus,.tubulang-input:hover{border-color:#1890ff}.tubulang-flex{display:flex}.tubulang-flex-align-center{align-items:center}.tubulang-flex-justify-center{justify-content:center}.tubulang-flex-column{flex-direction:column}.tubulang-button{height:32px;padding:4px 15px;border:1px solid #d9d9d9;background-color:#fff;cursor:pointer;font-size:14px;outline:0}.tubulang-button:active,.tubulang-button:focus,.tubulang-button:hover{color:#1890ff;border-color:#1890ff}.tubulang-button.tubulang-button-size-sm{height:24px;padding:0 7px;font-size:12px}.tubulang-button.tubulang-button-size-lg{height:40px;padding:6.4px 15px;font-size:16px}.tubulang-button.tubulang-button-type-primary{color:#fff;background-color:#1890ff}.tubulang-modal-root-hidden{display:none}.tubulang-modal-root .tubulang-modal-wrap{display:flex;position:fixed;justify-content:center;align-items:center;z-index:1000;top:0;left:0;right:0;bottom:0}.tubulang-modal-root .tubulang-modal-mask{z-index:1000;position:fixed;top:0;left:0;right:0;bottom:0;background-color:rgba(0,0,0,.45);height:100%}.tubulang-modal-root .tubulang-modal{background-color:#fff}.tubulang-modal-root .tubulang-modal.tubulang-modal-default{width:500px;height:400px}.tubulang-modal-root .tubulang-modal.tubulang-modal-lg{width:700px;height:500px}.tubulang-modal-root .tubulang-modal.tubulang-modal-sm{width:300px;height:200px}</style><meta name="generator" content="Gatsby 2.24.10"/><title data-rh="true">Input input | tubulang-ui</title><meta data-rh="true" name="description" content="My awesome app using docz"/><meta data-rh="true" property="og:title" content="Input input"/><meta data-rh="true" property="og:description" content="My awesome app using docz"/><meta data-rh="true" property="og:type" content="website"/><meta data-rh="true" name="twitter:card" content="summary"/><meta data-rh="true" name="twitter:title" content="Input input"/><meta data-rh="true" name="twitter:description" content="My awesome app using docz"/><link as="script" rel="preload" href="/lang-ui/webpack-runtime-2c205eb148494f3b60c2.js"/><link as="script" rel="preload" href="/lang-ui/framework-78f11fe170401ed1a198.js"/><link as="script" rel="preload" href="/lang-ui/styles-455b3516bc49bc6504de.js"/><link as="script" rel="preload" href="/lang-ui/c78d26b1-cb74bd766b7fb33aed98.js"/><link as="script" rel="preload" href="/lang-ui/app-dbbe749b34639890bd44.js"/><link as="script" rel="preload" href="/lang-ui/bac1b955-4ece88af0fb1f4362ec0.js"/><link as="script" rel="preload" href="/lang-ui/eb695efc1ed5f7f4f663d1db92480d3e9a50a712-416681de672fb5b6c1f4.js"/><link as="script" rel="preload" href="/lang-ui/component---src-components-input-index-mdx-03bae49a221e8597efd6.js"/><link as="fetch" rel="preload" href="/lang-ui/page-data/Input/page-data.json" crossorigin="anonymous"/><link as="fetch" rel="preload" href="/lang-ui/static/d/1635659820.json" crossorigin="anonymous"/><link as="fetch" rel="preload" href="/lang-ui/page-data/app-data.json" crossorigin="anonymous"/></head><body><div id="___gatsby"><div style="outline:none" tabindex="-1" id="gatsby-focus-wrapper"><style data-emotion-css="1ouyflx">.css-1ouyflx{font-family:'Source Sans Pro',sans-serif;font-size:20px;font-weight:400;line-height:1.5;color:var(--theme-ui-colors-text,#2D3747);background-color:var(--theme-ui-colors-background,#FFFFFF);}</style><div class="css-1ouyflx"><style data-emotion-css="jrjj6h">.css-jrjj6h > div{-webkit-flex:1 1 auto;-ms-flex:1 1 auto;flex:1 1 auto;}</style><style data-emotion-css="1ubpee3">.css-1ubpee3{min-height:100vh;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;}.css-1ubpee3 > div{-webkit-flex:1 1 auto;-ms-flex:1 1 auto;flex:1 1 auto;}</style><style data-emotion-css="hbri28">.css-hbri28{box-sizing:border-box;min-width:0;min-height:100vh;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;}.css-hbri28 > div{-webkit-flex:1 1 auto;-ms-flex:1 1 auto;flex:1 1 auto;}</style><div data-testid="layout" class="css-hbri28"><style data-emotion-css="1xva7m0">body{margin:0;padding:0;}.icon-link{display:none;}.with-overlay{overflow:hidden;}</style><style data-emotion-css="zf0iqh">.css-zf0iqh{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;min-height:100vh;}</style><style data-emotion-css="1tjxv7i">.css-1tjxv7i{-webkit-flex:1 1 auto;-ms-flex:1 1 auto;flex:1 1 auto;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;min-height:100vh;}</style><style data-emotion-css="238csb">.css-238csb{box-sizing:border-box;min-width:0;-webkit-flex:1 1 auto;-ms-flex:1 1 auto;flex:1 1 auto;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;min-height:100vh;}</style><main class="css-238csb"><style data-emotion-css="18wmtq">.css-18wmtq{background-color:var(--theme-ui-colors-header-bg,#F5F6F7);position:relative;z-index:1;border-bottom:1px solid var(--theme-ui-colors-border,#CED4DE);}</style><div data-testid="header" class="css-18wmtq"><style data-emotion-css="zankq9">.css-zankq9{display:none;position:absolute;top:calc(100% + 15px);left:30px;}@media screen and (max-width:57.5em){.css-zankq9{display:block;}}</style><style data-emotion-css="1s0wdqz">.css-1s0wdqz{box-sizing:border-box;min-width:0;display:none;position:absolute;top:calc(100% + 15px);left:30px;}@media screen and (max-width:57.5em){.css-1s0wdqz{display:block;}}</style><div class="css-1s0wdqz"><style data-emotion-css="4019lz">.css-4019lz{padding:0;outline:none;background:transparent;border:none;color:var(--theme-ui-colors-header-text,#2D3747);opacity:0.5;cursor:pointer;}.css-4019lz:hover{cursor:pointer;}</style><button class="css-4019lz"><svg xmlns="http://www.w3.org/2000/svg" width="25" height="25" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><line x1="3" y1="12" x2="21" y2="12"></line><line x1="3" y1="6" x2="21" y2="6"></line><line x1="3" y1="18" x2="21" y2="18"></line></svg></button></div><style data-emotion-css="z6jak6">.css-z6jak6{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;-webkit-box-pack:justify;-webkit-justify-content:space-between;-ms-flex-pack:justify;justify-content:space-between;padding-left:32px;padding-right:32px;position:relative;height:80px;}</style><div class="css-z6jak6"><style data-emotion-css="xenp3v">.css-xenp3v{-webkit-letter-spacing:-0.02em;-moz-letter-spacing:-0.02em;-ms-letter-spacing:-0.02em;letter-spacing:-0.02em;font-weight:600;font-size:24px;}</style><style data-emotion-css="1abzlu">.css-1abzlu{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-letter-spacing:-0.02em;-moz-letter-spacing:-0.02em;-ms-letter-spacing:-0.02em;letter-spacing:-0.02em;font-weight:600;font-size:24px;}</style><style data-emotion-css="ug0363">.css-ug0363{box-sizing:border-box;min-width:0;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-letter-spacing:-0.02em;-moz-letter-spacing:-0.02em;-ms-letter-spacing:-0.02em;letter-spacing:-0.02em;font-weight:600;font-size:24px;}</style><div data-testid="logo" class="css-ug0363"><style data-emotion-css="gyijir">.css-gyijir{font-weight:600;color:var(--theme-ui-colors-header-text,#2D3747);-webkit-text-decoration:none;text-decoration:none;}.css-gyijir:hover{color:var(--theme-ui-colors-primary,#0B5FFF);}</style><a class="css-gyijir" href="/lang-ui/">tubulang-ui</a></div><style data-emotion-css="k008qs">.css-k008qs{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;}</style><style data-emotion-css="80zs6q">.css-80zs6q{box-sizing:border-box;min-width:0;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;}</style><div class="css-80zs6q"><style data-emotion-css="1is33jg">.css-1is33jg{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;-webkit-box-pack:center;-webkit-justify-content:center;-ms-flex-pack:center;justify-content:center;outline:none;padding:12px;border:none;border-radius:9999px;background-color:var(--theme-ui-colors-header-button-bg,#0B5FFF);color:var(--theme-ui-colors-header-button-color,#FFFFFF);font-size:12px;font-weight:600;cursor:pointer;}</style><button aria-label="Switch to light mode" class="css-1is33jg"><svg xmlns="http://www.w3.org/2000/svg" width="15" height="15" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><circle cx="12" cy="12" r="5"></circle><line x1="12" y1="1" x2="12" y2="3"></line><line x1="12" y1="21" x2="12" y2="23"></line><line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line><line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line><line x1="1" y1="12" x2="3" y2="12"></line><line x1="21" y1="12" x2="23" y2="12"></line><line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line><line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line></svg></button></div></div></div><style data-emotion-css="1nfi0ai">.css-1nfi0ai{padding-top:0;padding-bottom:0;-webkit-flex:1;-ms-flex:1;flex:1;display:grid;grid-template-columns:250px minmax(0,1fr);min-height:100vh;}@media screen and (max-width:57.5em){.css-1nfi0ai{display:block;}}</style><div class="css-1nfi0ai"><style data-emotion-css="1tn7q1x">.css-1tn7q1x{z-index:999;position:fixed;top:81px;right:0;bottom:0;left:0;background:rgba(0,0,0,0.6);-webkit-transition:all .2s ease-out;transition:all .2s ease-out;visibility:hidden;opacity:0;}</style><style data-emotion-css="h4snl3">.css-h4snl3{box-sizing:border-box;min-width:0;z-index:999;position:fixed;top:81px;right:0;bottom:0;left:0;background:rgba(0,0,0,0.6);-webkit-transition:all .2s ease-out;transition:all .2s ease-out;visibility:hidden;opacity:0;}</style><div class="css-h4snl3"></div><style data-emotion-css="1ethd5x">.css-1ethd5x{padding-top:32px;padding-bottom:32px;padding-left:32px;padding-right:32px;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;position:-webkit-sticky;position:sticky;top:0;z-index:1;min-width:0;max-height:100vh;border-right:1px solid var(--theme-ui-colors-border,#CED4DE);overflow:auto;-webkit-overflow-scrolling:touch;background-color:var(--theme-ui-colors-sidebar-bg,#FFFFFF);}@media screen and (max-width:57.5em){.css-1ethd5x{z-index:9999;display:block;position:fixed;top:81px;left:0;bottom:0;width:256px;padding-left:32px;padding-right:32px;background-color:var(--theme-ui-colors-background,#FFFFFF);-webkit-transition:-webkit-transform .2s ease-out;-webkit-transition:transform .2s ease-out;transition:transform .2s ease-out;-webkit-transform:translateX(-100%);-ms-transform:translateX(-100%);transform:translateX(-100%);}}</style><style data-emotion-css="1li9yso">.css-1li9yso{box-sizing:border-box;min-width:0;padding-top:32px;padding-bottom:32px;padding-left:32px;padding-right:32px;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;position:-webkit-sticky;position:sticky;top:0;z-index:1;min-width:0;max-height:100vh;border-right:1px solid var(--theme-ui-colors-border,#CED4DE);overflow:auto;-webkit-overflow-scrolling:touch;background-color:var(--theme-ui-colors-sidebar-bg,#FFFFFF);}@media screen and (max-width:57.5em){.css-1li9yso{z-index:9999;display:block;position:fixed;top:81px;left:0;bottom:0;width:256px;padding-left:32px;padding-right:32px;background-color:var(--theme-ui-colors-background,#FFFFFF);-webkit-transition:-webkit-transform .2s ease-out;-webkit-transition:transform .2s ease-out;transition:transform .2s ease-out;-webkit-transform:translateX(-100%);-ms-transform:translateX(-100%);transform:translateX(-100%);}}</style><div data-testid="sidebar" class="css-1li9yso"><style data-emotion-css="lcm9hh">.css-lcm9hh{margin-bottom:16px;display:-webkit-inline-box;display:-webkit-inline-flex;display:-ms-inline-flexbox;display:inline-flex;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;}</style><div data-testid="nav-search" class="css-lcm9hh"><style data-emotion-css="1mo6uhh">.css-1mo6uhh{color:var(--theme-ui-colors-border,#CED4DE);margin-right:8px;}</style><svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="css-1mo6uhh"><circle cx="11" cy="11" r="8"></circle><line x1="21" y1="21" x2="16.65" y2="16.65"></line></svg><style data-emotion-css="1bbueiu">.css-1bbueiu{outline:none;background:none;border:none;color:var(--theme-ui-colors-text,#2D3747);font-size:14px;}</style><input placeholder="Type to search..." value="" class="css-1bbueiu"/></div><style data-emotion-css="1ebprri">.css-1ebprri{margin-top:16px;margin-bottom:16px;}</style><div data-testid="nav-group" class="css-1ebprri"><style data-emotion-css="1c8kbwa">.css-1c8kbwa{margin-bottom:4px;font-size:16px;font-weight:500;color:var(--theme-ui-colors-sidebar-navGroup,#2D3747);cursor:pointer;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-pack:justify;-webkit-justify-content:space-between;-ms-flex-pack:justify;justify-content:space-between;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;}</style><div class="css-1c8kbwa">组件<style data-emotion-css="1scp5x8">.css-1scp5x8{margin-left:4px;-webkit-flex-shrink:0;-ms-flex-negative:0;flex-shrink:0;-webkit-align-self:baseline;-ms-flex-item-align:baseline;align-self:baseline;-webkit-transform:rotateX(180deg);-ms-transform:rotateX(180deg);transform:rotateX(180deg);-webkit-transform-origin:center;-ms-transform-origin:center;transform-origin:center;-webkit-transition:-webkit-transform .3s ease-in-out;-webkit-transition:transform .3s ease-in-out;transition:transform .3s ease-in-out;}</style><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="css-1scp5x8"><polyline points="6 9 12 15 18 9"></polyline></svg></div><style data-emotion-css="1isemmb">.css-1isemmb{margin-left:8px;}</style><div data-testid="nav-group-links" class="css-1isemmb"><style data-emotion-css="1rqvyud">.css-1rqvyud{margin-top:8px;margin-bottom:8px;display:block;color:var(--theme-ui-colors-sidebar-navGroup,#2D3747);-webkit-text-decoration:none;text-decoration:none;font-size:16px;}.css-1rqvyud.active{color:var(--theme-ui-colors-sidebar-navLinkActive,#0B5FFF);}</style><a class="css-1rqvyud" href="/lang-ui/Alert">Alert 警告提示</a><a class="css-1rqvyud" href="/lang-ui/Button">Button 按钮</a><a class="css-1rqvyud" href="/lang-ui/Flex">Flex flex布局</a><a aria-current="page" class="css-1rqvyud active" href="/lang-ui/Input">Input input</a><style data-emotion-css="1ooej63">.css-1ooej63{margin-top:8px;margin-bottom:8px;display:block;color:var(--theme-ui-colors-sidebar-tocLink,#67788a);-webkit-text-decoration:none;text-decoration:none;font-size:14px;margin-left:16px;position:relative;}.css-1ooej63.active{color:var(--theme-ui-colors-sidebar-tocLinkActive,#1D2330);}.css-1ooej63.active::before{content:"";position:absolute;display:block;top:2px;left:-8px;height:1rem;background-color:var(--theme-ui-colors-primary,#0B5FFF);-webkit-transition:width 200ms ease 0s;transition:width 200ms ease 0s;width:2px;border-radius:1px;}</style><a class=" css-1ooej63" href="/lang-ui/Input#代码演示">代码演示</a><a class=" css-1ooej63" href="/lang-ui/Input#api">API</a><a class="css-1rqvyud" href="/lang-ui/InputNumber">InputNumber 数字输入框</a><a class="css-1rqvyud" href="/lang-ui/Modal">Modal 模态框</a><a class="css-1rqvyud" href="/lang-ui/Portal">Portal 传送门</a></div></div></div><style data-emotion-css="1w3d2mg">.css-1w3d2mg{background-color:var(--theme-ui-colors-background,#FFFFFF);position:relative;max-width:1280px;padding-top:48px;padding-bottom:48px;padding-left:32px;padding-right:32px;padding:32px;}@media screen and (max-width:57.5em){.css-1w3d2mg{padding-top:48px;padding-bottom:32px;padding-left:32px;padding-right:32px;}}</style><style data-emotion-css="kjtkh5">.css-kjtkh5{width:100%;min-width:0;max-width:1280px;margin-left:auto;margin-right:auto;padding:32px;background-color:var(--theme-ui-colors-background,#FFFFFF);position:relative;max-width:1280px;padding-top:48px;padding-bottom:48px;padding-left:32px;padding-right:32px;padding:32px;}@media screen and (max-width:57.5em){.css-kjtkh5{padding-top:48px;padding-bottom:32px;padding-left:32px;padding-right:32px;}}</style><style data-emotion-css="1307vk3">.css-1307vk3{box-sizing:border-box;min-width:0;width:100%;min-width:0;max-width:1280px;margin-left:auto;margin-right:auto;padding:32px;background-color:var(--theme-ui-colors-background,#FFFFFF);position:relative;max-width:1280px;padding-top:48px;padding-bottom:48px;padding-left:32px;padding-right:32px;padding:32px;}@media screen and (max-width:57.5em){.css-1307vk3{padding-top:48px;padding-bottom:32px;padding-left:32px;padding-right:32px;}}</style><div data-testid="main-container" class="css-1307vk3"><style data-emotion-css="1vci3bl">.css-1vci3bl{padding:0;margin:0;margin-bottom:16px;font-size:48px;font-family:'Source Sans Pro',sans-serif;line-height:1.125;font-weight:700;}</style><h1 id="input-input" class="css-1vci3bl">Input input</h1><h2 id="代码演示"><style data-emotion-css="1r0pe6c">.css-1r0pe6c{color:inherit;-webkit-text-decoration:none;text-decoration:none;}.css-1r0pe6c:hover{-webkit-text-decoration:underline;text-decoration:underline;}</style><a href="#代码演示" class="css-1r0pe6c">代码演示</a></h2><h3 id="基本使用"><a href="#基本使用" class="css-1r0pe6c">基本使用</a></h3><div class="sc-AxirZ bJCmFu"><section class="code-box-demo"> <input class="tubulang-input"/></section><section class="code-box-meta"><div class="text-divider"><span>示例</span></div><div class="code-box-description"><p>暂无描述</p></div><div class="divider"></div><div class="code-box-action"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" data-place="top" data-tip="复制代码"><rect x="9" y="9" width="13" height="13" rx="2" ry="2"></rect><path d="M5 15H4a2 2 0 0 1-2-2V4a2 2 0 0 1 2-2h9a2 2 0 0 1 2 2v1"></path></svg><div class="sc-AxjAm bcMPWx"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" data-place="top" data-tip="显示代码"><polyline points="16 18 22 12 16 6"></polyline><polyline points="8 6 2 12 8 18"></polyline></svg></div></div></section><div class="__react_component_tooltip tefd0a60e-cef6-4d38-90b7-2aae6236d28e place-top type-dark" data-id="tooltip"><style>
  	.tefd0a60e-cef6-4d38-90b7-2aae6236d28e {
	    color: #fff;
	    background: #222;
	    border: 1px solid transparent;
  	}

  	.tefd0a60e-cef6-4d38-90b7-2aae6236d28e.place-top {
        margin-top: -10px;
    }
    .tefd0a60e-cef6-4d38-90b7-2aae6236d28e.place-top::before {
        border-top: 8px solid transparent;
    }
    .tefd0a60e-cef6-4d38-90b7-2aae6236d28e.place-top::after {
        border-left: 8px solid transparent;
        border-right: 8px solid transparent;
        bottom: -6px;
        left: 50%;
        margin-left: -8px;
        border-top-color: #222;
        border-top-style: solid;
        border-top-width: 6px;
    }

    .tefd0a60e-cef6-4d38-90b7-2aae6236d28e.place-bottom {
        margin-top: 10px;
    }
    .tefd0a60e-cef6-4d38-90b7-2aae6236d28e.place-bottom::before {
        border-bottom: 8px solid transparent;
    }
    .tefd0a60e-cef6-4d38-90b7-2aae6236d28e.place-bottom::after {
        border-left: 8px solid transparent;
        border-right: 8px solid transparent;
        top: -6px;
        left: 50%;
        margin-left: -8px;
        border-bottom-color: #222;
        border-bottom-style: solid;
        border-bottom-width: 6px;
    }

    .tefd0a60e-cef6-4d38-90b7-2aae6236d28e.place-left {
        margin-left: -10px;
    }
    .tefd0a60e-cef6-4d38-90b7-2aae6236d28e.place-left::before {
        border-left: 8px solid transparent;
    }
    .tefd0a60e-cef6-4d38-90b7-2aae6236d28e.place-left::after {
        border-top: 5px solid transparent;
        border-bottom: 5px solid transparent;
        right: -6px;
        top: 50%;
        margin-top: -4px;
        border-left-color: #222;
        border-left-style: solid;
        border-left-width: 6px;
    }

    .tefd0a60e-cef6-4d38-90b7-2aae6236d28e.place-right {
        margin-left: 10px;
    }
    .tefd0a60e-cef6-4d38-90b7-2aae6236d28e.place-right::before {
        border-right: 8px solid transparent;
    }
    .tefd0a60e-cef6-4d38-90b7-2aae6236d28e.place-right::after {
        border-top: 5px solid transparent;
        border-bottom: 5px solid transparent;
        left: -6px;
        top: 50%;
        margin-top: -4px;
        border-right-color: #222;
        border-right-style: solid;
        border-right-width: 6px;
    }
  </style></div></div><h2 id="api"><a href="#api" class="css-1r0pe6c">API</a></h2><style data-emotion-css="134anpw">.css-134anpw{padding:0;margin:0;margin-bottom:32px;border-collapse:separate;width:100%;margin-top:32px;border-spacing:0;}.css-134anpw th,.css-134anpw td{text-align:left;padding-top:4px;padding-bottom:4px;padding-right:4px;padding-left:0;border-color:var(--theme-ui-colors-muted,#67788a);border-bottom-style:solid;}</style><table class="css-134anpw"><thead><tr class="css-0"><style data-emotion-css="hafajd">.css-hafajd{text-align:left;border-bottom:1px solid;padding-left:8px;padding-right:8px;padding-top:4px;padding-bottom:4px;vertical-align:bottom;border-bottom-width:2px;}.css-hafajd:first-child{padding-left:0;}.css-hafajd:last-child{padding-right:0;}</style><th class="css-hafajd">属性</th><th class="css-hafajd">说明</th><th class="css-hafajd">类型</th><th class="css-hafajd">默认值</th></tr></thead><tbody></tbody></table></div></div></main></div></div></div><div id="gatsby-announcer" style="position:absolute;top:0;width:1px;height:1px;padding:0;overflow:hidden;clip:rect(0, 0, 0, 0);white-space:nowrap;border:0" aria-live="assertive" aria-atomic="true"></div></div><script id="gatsby-script-loader">/*<![CDATA[*/window.pagePath="/Input";/*]]>*/</script><script id="gatsby-chunk-mapping">/*<![CDATA[*/window.___chunkMapping={"polyfill":["/polyfill-79eb8eced54fc21002d7.js"],"app":["/app-dbbe749b34639890bd44.js"],"component---src-components-alert-index-mdx":["/component---src-components-alert-index-mdx-4380cfa12ce199c7798d.js"],"component---src-components-button-index-mdx":["/component---src-components-button-index-mdx-a712330d70513b85f53e.js"],"component---src-components-flex-index-mdx":["/component---src-components-flex-index-mdx-87c2372466600ab7cfa8.js"],"component---src-components-input-index-mdx":["/component---src-components-input-index-mdx-03bae49a221e8597efd6.js"],"component---src-components-input-number-index-mdx":["/component---src-components-input-number-index-mdx-9b108f827de51122616f.js"],"component---src-components-modal-index-mdx":["/component---src-components-modal-index-mdx-2977a328a61e1fabec6d.js"],"component---src-components-portal-index-mdx":["/component---src-components-portal-index-mdx-b7cfb5fb910155993df8.js"],"component---src-pages-404-js":["/component---src-pages-404-js-ca759bdd496aef2620de.js"]};/*]]>*/</script><script src="/lang-ui/polyfill-79eb8eced54fc21002d7.js" nomodule=""></script><script src="/lang-ui/component---src-components-input-index-mdx-03bae49a221e8597efd6.js" async=""></script><script src="/lang-ui/eb695efc1ed5f7f4f663d1db92480d3e9a50a712-416681de672fb5b6c1f4.js" async=""></script><script src="/lang-ui/bac1b955-4ece88af0fb1f4362ec0.js" async=""></script><script src="/lang-ui/app-dbbe749b34639890bd44.js" async=""></script><script src="/lang-ui/c78d26b1-cb74bd766b7fb33aed98.js" async=""></script><script src="/lang-ui/styles-455b3516bc49bc6504de.js" async=""></script><script src="/lang-ui/framework-78f11fe170401ed1a198.js" async=""></script><script src="/lang-ui/webpack-runtime-2c205eb148494f3b60c2.js" async=""></script></body></html>