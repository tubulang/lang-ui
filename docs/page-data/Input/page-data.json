{"componentChunkName":"component---src-components-input-index-mdx","path":"/Input","result":{"pageContext":{"frontmatter":{"name":"Input input","route":"/Input","menu":"组件"},"entry":{"id":"5ae04a6e9f1270cdc395b76e3410e2f1","filepath":"src/components/input/index.mdx","fullpath":"/Users/tubulang/personalProject/tubulang-ui/src/components/input/index.mdx","route":"/Input","slug":"src-components-input-index","name":"Input input","menu":"组件","headings":[{"slug":"input-input","depth":1,"value":"Input input"},{"slug":"代码演示","depth":2,"value":"代码演示"},{"slug":"基本使用","depth":3,"value":"基本使用"},{"slug":"api","depth":2,"value":"API"}]}}},"staticQueryHashes":["1635659820"]}