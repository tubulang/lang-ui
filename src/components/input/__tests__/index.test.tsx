import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import Input from '../index'

describe('Input', () => {
  it('renders without error', () => {
    const { container } = render(<Input />)
    expect(container).toMatchSnapshot()
  })
})
