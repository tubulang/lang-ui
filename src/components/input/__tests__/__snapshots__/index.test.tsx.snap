// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input renders without error 1`] = `
<div>
  <input
    class="tubulang-input"
  />
</div>
`;
